platform/engineering/design-docs/separate-content-build.md

```diff
@@ -0,0 +1,477 @@
+# Separate Content Build
```

Comment:
filename: can you rename this to something with a YYYY-MM-DD date?

i.e. 2020-03-31-separate-content-build.md